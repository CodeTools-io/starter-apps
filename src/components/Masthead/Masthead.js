import React from 'react';

import { Box, Image, Stack } from 'grommet';
export default function Masthead({
  imageSrc,
  contentLocation = 'center',
  children,
}) {
  return (
    <Box className="Masthead">
      <Stack anchor={contentLocation} fill>
        <Image src={imageSrc} fit="cover" fill />
        {children}
      </Stack>
    </Box>
  );
}
