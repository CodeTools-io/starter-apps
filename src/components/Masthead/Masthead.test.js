import React from 'react'
import Masthead from './Masthead'

describe('Masthead', () => {
  test('can render', () => {
    const { container } = renderComponent(
      <Masthead>
        <p>Content for the Masthead</p>
      </Masthead>
    )

    expect(container).toMatchSnapshot()
  })
})

