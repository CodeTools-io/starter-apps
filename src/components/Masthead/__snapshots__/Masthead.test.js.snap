// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Masthead can render 1`] = `
<div>
  <div
    class="StyledBox-sc-13pk1d4-0 bFwQzJ Masthead"
  >
    <div
      class="StyledStack-ajspsk-0 KTKTv"
    >
      <div
        class="StyledStack__StyledStackLayer-ajspsk-1 bEYrkt"
      >
        <img
          class="StyledImage-ey4zx9-0 kdPbXO"
        />
      </div>
      <div
        class="StyledStack__StyledStackLayer-ajspsk-1 ddIVvW"
      >
        <p>
          Content for the Masthead
        </p>
      </div>
    </div>
  </div>
</div>
`;
